src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculate(double first, double second, char operation) {
+        switch (operation) {
+            case '+' :
```

Comment:
Краще винести в окреме поле, 
```suggestion
            private static final char ADDITION =  '+' ;  // в класі
            case ADDITION :
```

Reply:
і так кожну операцію